Reverts:
------------------------------------------------------------
revno: 239 [merge]
author: Charles Kerr <charles.kerr@canonical.com>
committer: Tarmac
branch nick: trunk.13.04
timestamp: Wed 2013-01-23 07:49:19 +0000
message:
  remove g_type_init() calls; introduce explicit glib requirements to configure.ac and debian/control
  
  submitting for Jenkins to test b/c I haven't upgraded my glib yet. Fixes: https://bugs.launchpad.net/bugs/1103087.
  
  Approved by Lars Uebernickel, PS Jenkins bot.
------------------------------------------------------------
=== modified file 'configure.ac'
--- configure.ac	2013-01-22 20:32:18 +0000
+++ configure.ac	2012-08-22 11:59:58 +0000
@@ -29,7 +29,6 @@
 # Dependencies
 ###########################
 
-GLIB_REQUIRED_VERSION=2.35.4
 GTK_REQUIRED_VERSION=3.0.0
 GIO_REQUIRED_VERSION=2.26
 INDICATOR_REQUIRED_VERSION=0.3.5
@@ -39,9 +38,8 @@
 
 LIBAPPINDICATOR_REQUIRED_VERSION=0.2.9
 
-PKG_CHECK_MODULES(INDICATOR,  glib-2.0 >= $GLIB_REQUIRED_VERSION
+PKG_CHECK_MODULES(INDICATOR,  gtk+-3.0 >= $GTK_REQUIRED_VERSION
                               gio-2.0 >= $GIO_REQUIRED_VERSION
-                              gtk+-3.0 >= $GTK_REQUIRED_VERSION
                               indicator3-0.4 >= $INDICATOR_REQUIRED_VERSION
                               json-glib-1.0 >= $JSON_GLIB_REQUIRED_VERSION
                               dbus-glib-1 >= $DBUS_GLIB_REQUIRED_VERSION

=== modified file 'src/application-service.c'
--- src/application-service.c	2013-01-22 20:32:18 +0000
+++ src/application-service.c	2012-02-04 02:34:52 +0000
@@ -52,6 +52,8 @@
 int
 main (int argc, char ** argv)
 {
+	g_type_init();
+
 	/* Bring us up as a basic indicator service */
 	service = indicator_service_new_version(INDICATOR_APPLICATION_DBUS_ADDR, INDICATOR_APPLICATION_SERVICE_VERSION);
 	g_signal_connect(G_OBJECT(service), INDICATOR_SERVICE_SIGNAL_SHUTDOWN, G_CALLBACK(service_disconnected), NULL);

